t/17-labels_carry_over.t

```diff
@@ -33,6 +34,7 @@ my $t         = Test::Mojo->new('OpenQA::WebAPI');
 my $rs        = $t->app->schema->resultset('Jobs');
 my $auth      = {'X-CSRF-Token' => $t->ua->get('/tests')->res->dom->at('meta[name=csrf-token]')->attr('content')};
 $test_case->login($t, 'percival');
+assume_all_assets_exist;
```

Comment:
what do these assets have to do with labels carry over? Does the test fail if we don't call this?

Reply:
This test restarts jobs and that's prevented by default if assets are missing. So yes, the test fails if we don't call this. I only added this call to tests which would otherwise fail.

Reply:
ok, cool